### ISSUE TITLE
Set the default value of useShortNames to `true`

### ISSUE THREAD

**Issue by tianyiw2013** [neild3r/vscode-php-docblocker#178]
Now PHP's built-in integer and boolean are `int` and `bool`.
I think we should use the same type as PHP.

What's your opinion?

**Comment by neild3r** [comment]
Agreed, I use short names myself I only haven't changed the default as I didn't want to release a breaking change. I suppose we could do it as part of a move to v3
